### DISCUSSION TITLE
What do the `a` and `d` sort keys mean?

### DISCUSSION THREAD

**Discussion by azuline** [jarun/nnn#1750] [Q&A] [answered]
The manpage has:

```
       -T key
               sort order
               keys: 'a'u / 'd'u / 'e'xtension / 'r'everse / 's'ize / 't'ime / 'v'ersion
               capitalize to reverse (except 'r')
```

and `erstv` are pretty self-explanatory, but what do `au` and `du` mean? I've tried playing with these sorts, but I can't figure out what they mean.

**Comment by azuline** [comment, marked as answer]
Never mind; I forgot that I can read code. The answer seems to be apparent size (disk usage) and blocks used (disk usage).

**Reply by raffaelemancuso** [reply]
@jarun @N-R-K can we write that in the wiki please?

**Reply by N-R-K** [reply]
> can we write that in the wiki please?

Feel free to open a PR and make it clear in the manpage directly.

**Reply by jarun** [reply]
@raffaem it's there in the wiki. See the notes section: https://github.com/jarun/nnn/wiki/Usage#program-options

Also, `du` is a very common term since the utility `du` is named that way.

**Reply by raffaelemancuso** [reply]
@N-R-K sorry, now I don't understand how the 's' sort option differs from 'a' and 'd'

**Reply by N-R-K** [reply]
> I don't understand how the 's' sort option differs from 'a' and 'd'

au and du descends into dirs. 's'ize refers to file size and won't recursively go inside of dirs.
